src/MarkerLineChart.js

```diff
@@ -254,9 +276,21 @@ export default class MarkerLineChart extends React.Component {
     if (!_.every([x1, x2, y1, y2], _.isFinite)) return null;
     return (
       <line
-        className="marker-line"
+        className={`${getValue(lineClassName, d, i)}`}
+        style={getValue(lineStyle, d, i)}
         {...{ x1, x2, y1, y2, key, onMouseEnter, onMouseMove, onMouseLeave }}
       />
     );
   };
+
+  render() {
+    const tickType = getTickType(this.props);
+    return (
+      <g className="rct-marker-line-chart">
+        {tickType === "RangeValue"
```

Comment:
is `RangeValue` something used across multiple files, and if so should this string be abstracted into a constants file?

Reply:
on inspection it's a tickType conceptually used only for MarkerLineChart